app/renderer/components/navigation/urlBar.js

```diff
@@ -481,7 +482,23 @@ class UrlBar extends React.Component {
     return props
   }
 
+  get showEvCert () {
+    return !this.props.titleMode && <span className='evCert'> {this.props.evCert} </span>
```

Comment:
this will return `false` if we are in titleMode. according to react docs (https://reactjs.org/docs/conditional-rendering.html#preventing-component-from-rendering), it should return `null` when an element isn't rendered.

Reply:
thanks changed in 97f689b6a7c12ca0ee009bdfaf514706e33c0b1a via `push -f`